src/main/pg/bus_spi.c

```diff
@@ -107,14 +107,8 @@ ioTag_t preinitIPUList[SPI_PREINIT_IPU_COUNT] = {
 #ifdef MS5611_CS_PIN
     IO_TAG(MS5611_CS_PIN),
 #endif
-#ifdef LPS_CS_PIN
-    IO_TAG(LPS_CS_PIN),
-#endif
-#ifdef HMC5883_CS_PIN
-    IO_TAG(HMC5883_CS_PIN),
-#endif
-#ifdef AK8963_CS_PIN
-    IO_TAG(AK8963_CS_PIN),
+#ifdef MAG_CS_PIN
+    IO_TAG(MAG_CS_PIN),
```

Comment:
I think this should be made to work with `compassConfig->mag_spi_csn`, or else it will become nonfunctional for universal targets.

Reply:
The Gyro driver is working here is working in the same way? BTW this is already tested for the generic target in PR #6805.

Reply:
I suspect the other drivers will have to be changed in the same way. Fact is this will not do anything in the generic target since `MAG_CS_PIN` is `NONE` there. If not doing anything here is acceptable then it's best to simplify be removing the code that only works for pre-universal-targets, and simplify debugging.

Reply:
Condition already handled in #6870.

Reply:
See my comment on #6870. This is broken even for non - generic targets.